src/lib/mpq.mjs

```diff
@@ -18,6 +18,39 @@ class MPQ {
     }
   }
 
+  find(mask, listfile = '') {
+    this._ensureHandle();
+
+    const findData = new StormLib.SFileFindData();
+
+    const findHandle = StormLib.SFileFindFirstFile(this.handle, mask, findData, listfile);
+
+    if (findHandle.isNull()) {
+      const errno = StormLib.GetLastError();
+
+      if (errno === StormLib.ERROR_NO_MORE_FILES) {
+        return [];
+      } else {
+        throw new Error(`Find failed (error ${errno})`);
+      }
+    }
```

Comment:
Ah neat! I think this was actually missing from Blizzardry, or?

Reply:
Yeah! I think Blizzardry returns an empty array in all cases where the find functions fail. One of the things I'm loving about `embind` is how easy it is to make constants from the headers available in JS. I just mark them in the bindings and voila, they're available on `StormLib`.

Reply:
Ah that's super great indeed 👍